package.json

```diff
@@ -58,16 +59,20 @@
     "@babel/runtime": "7.14.0",
     "@babylonjs/core": "4.1.0",
     "@babylonjs/loaders": "4.1.0",
+    "assert": "2.0.0",
+    "buffer": "6.0.3",
     "classnames": "2.3.1",
     "codesandbox": "2.2.3",
     "oimo": "1.0.9",
+    "path-browserify": "1.0.1",
     "prism-react-renderer": "1.2.0",
+    "process": "^0.11.10",
```

Comment:
All these new dependencies are needed for `import {parse} from 'react-docgen'` to work